File: CS/GridProject/ExtendedGridView/ExtendedGridViewInfoRegistrator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraGrid.Registrator;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid;

namespace ExtendedGridViewSpace
{
    public class ExtendedGridViewInfoRegistrator : GridInfoRegistrator
    {
        public override string ViewName { get { return "ExtendedGridView"; } }

        public override BaseView CreateView(GridControl grid)
        {
            return new ExtendedGridView(grid as GridControl);
        }

    }
}

File: VB/GridProject/ExtendedGridView/ExtendedGridViewInfoRegistrator.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraGrid.Registrator
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid

Namespace ExtendedGridViewSpace
	Public Class ExtendedGridViewInfoRegistrator
		Inherits GridInfoRegistrator
		Public Overrides ReadOnly Property ViewName() As String
			Get
				Return "ExtendedGridView"
			End Get
		End Property

		Public Overrides Function CreateView(ByVal grid As GridControl) As BaseView
			Return New ExtendedGridView(TryCast(grid, GridControl))
		End Function

	End Class
End Namespace
